src/hotspot/cpu/riscv/stubGenerator_riscv.cpp

```diff
@@ -2276,6 +2276,171 @@ class StubGenerator: public StubCodeGenerator {
     StubRoutines::_arrayof_jint_fill = generate_fill(T_INT, true, "arrayof_jint_fill");
   }
 
+  void generate_aes_loadkeys(const Register &key, VectorRegister *working_vregs, int rounds) {
+    const int step = 16;
+    for (int i = 0; i < rounds; i++) {
+      __ vle32_v(working_vregs[i], key);
+      __ vrev8_v(working_vregs[i], working_vregs[i]);
```

Comment:
What's is this `vrev8_v` for? I wonder how this maps to the code-samples at [1]. Might deserve a code comment here. 

[1] https://github.com/riscv/riscv-crypto/blob/main/doc/vector/code-samples/zvkned.s

Reply:
I'm not very familiar with OpenSSL but I guess it is just uses different endianness when storing the result :shrug:  (compared to [big-endian](https://docs.oracle.com/javase/specs/jvms/se23/html/jvms-2.html#jvms-2.11) JVM). I'll left the note on it in the code

Reply:
Hmm ... I didn't find a direct evidence about the endianness of the key required here. Was is documented anywhere? I prefer to add some more details in the code comment to make it more readable.

Reply:
All I was able to find is [this](https://stackoverflow.com/a/9455607) answer :shrug: Well if it is fully big-endian and in our case we store keys little-endian (as stated in the comment [here](https://github.com/openjdk/jdk/pull/19960/files#diff-97f199af6d1c8c17b2fa4f50eb1bbc0081858cc59a899f32792a2d31f933ccc4R2304)) then the difference in keys loading is expected (but again, it is just the guess :confused:). I'll expand the comment, no problem!